Sources/KituraWebSocket/WebSocketConnection.swift

```diff
@@ -196,7 +199,8 @@ public class WebSocketConnection {
     }
     
     func received(frame: WSFrame) {
-        
+        lastFrameReceivedAt = Date()
```

Comment:
Would it make sense to set the lastFrameReceivedAt inside `case .pong:` on line 289? This would allow you to still use pings to check if a connection is alive without affecting other received frames.

Reply:
Thank you @bridger for raising this pull request. I have just one comment regarding when `lastFrameReceivedAt` is set. Otherwise I think this is a good idea to allow clean up of stale connections.

Reply:
If we make this change `lastFrameReceivedAt` should probably be changed to something like 'lastPongReceivedAt`.